### PR TITLE
Feature info anzeige

### PR DESCRIPTION
**Pull request by SchulA**
Creation of configurable feature info display

### CODE REVIEW

File: src/components/ToolMenu/FeatureInfo/FeatureInfoTabs/index.tsx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    if (!vectorLayer) {
+      return;
+    }
+
+    vectorLayer.getSource()?.clear();
+    vectorLayer.getSource()?.addFeature(selectedFeature);
+  }, [selectedFeature, map, vectorLayerName]);
+
+  const onChange = (page: number) => {
+    setCurrentPage(page);
```
Comment (LukasLohoff) [inline src/components/ToolMenu/FeatureInfo/FeatureInfoTabs/index.tsx:93]: you could add a check here to make sure `page` is never smaller than 1

File: src/components/ToolMenu/FeatureInfo/FeatureInfoTabs/index.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    setSelectedFeature(features[page - 1]);
+  };
+
+  if (!tabConfig || !selectedFeature) {
+    return <></>;
+  }
+
+  const exportFilter = (propertyName: string) => {
```
Comment (LukasLohoff) [inline src/components/ToolMenu/FeatureInfo/FeatureInfoTabs/index.tsx:101]: just as a hint for the future, it should also be possible to find out if any `propertyName` in the array matches with `Array.some()`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

but this is also absolutely fine and should work just as well and doesn't have to be changed

**Review by LukasLohoff — APPROVED** [review]
Very nice @SchulA !

Left some smaller suggestions above.

File: src/components/ToolMenu/FeatureInfo/DisplayField/index.tsx
[the remark this answers is not in the document]
Comment (dnlkoch) [reply src/components/ToolMenu/FeatureInfo/DisplayField/index.tsx:1]: Nice catch, fixed in a5d59fc.

File: src/components/ToolMenu/FeatureInfo/FeatureInfoForm/index.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+export const FeatureInfoForm: React.FC<FeatureInfoFormProps> = ({
+  formConfig,
+  feature,
+  ...passThroughProps
+}): JSX.Element => {
+
+  const [form] = useForm();
+
+  useEffect(() => {
+    form.setFieldsValue(feature.getProperties());
```
[the remark this answers is not in the document]
Comment (dnlkoch) [reply src/components/ToolMenu/FeatureInfo/FeatureInfoForm/index.tsx:36]: According to the type of `useForm()`, `form` should always be defined.

File: src/components/ToolMenu/FeatureInfo/FeatureInfoTabs/index.tsx (hunk 1 above)
Comment (dnlkoch) [reply src/components/ToolMenu/FeatureInfo/FeatureInfoTabs/index.tsx:93]: Added check in 5b12a04.

File: src/components/ToolMenu/FeatureInfo/FeatureInfoForm/index.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (LukasLohoff) [reply src/components/ToolMenu/FeatureInfo/FeatureInfoForm/index.tsx:36]: ok, then nevermind!

**Review by LukasLohoff — APPROVED** [review]

**Comment by dnlkoch** [comment]
Thanks for the review!

**Pull request merged**